### ISSUE TITLE
debian jessie install issue

### ISSUE THREAD

**Issue by rgevaert** [broadinstitute/puppet-amavisd#1]
Hi

Installation on Debian jessie fails.  install.pp creates the amavis user, but the homedirectory isn't created.  Now it seems that in de postinst script it checks if the homedir does exist.

What would the problem be with setting managehome to false?

```
Info: Applying configuration version '1473941419'
Error: Execution of '/usr/bin/apt-get -q -y -o DPkg::Options::=--force-confold install amavisd-new' returned 100: Reading package lists...
Building dependency tree...
Reading state information...
amavisd-new is already the newest version.
The following packages were automatically installed and are no longer required:
  augeas-lenses libaugeas0 libconfig-tiny-perl libmath-calc-units-perl
  libmodule-implementation-perl libmodule-runtime-perl libnagios-plugin-perl
  libparams-classify-perl libparams-validate-perl libtry-tiny-perl
  libxml-dom-perl libxml-perl libxml-regexp-perl libxml-xpath-perl
  monitoring-plugins monitoring-plugins-standard nagios-nrpe-server
  nagios-plugins ruby ruby-augeas ruby-hiera ruby-json ruby-safe-yaml
  ruby-shadow ruby2.1 rubygems-integration
Use 'apt-get autoremove' to remove them.
0 upgraded, 0 newly installed, 0 to remove and 4 not upgraded.
1 not fully installed or removed.
After this operation, 0 B of additional disk space will be used.
Setting up amavisd-new (1:2.10.1-2~deb8u1) ...
+ echo Creating/updating amavis user account...
Creating/updating amavis user account...
+ getent group amavis
+ getent passwd amavis
+ getent+  passwdcut -d : -f 6
 amavis
+ homedir=/var/lib/amavis
+ [ x/var/lib/amavis = x/ ]
+ [ ! -d /var/lib/amavis ]
+ echo Broken amavis system user found.  To insure data-safety,
Broken amavis system user found.  To insure data-safety,
+ echo it is up to you the local administrator to remove the broken
it is up to you the local administrator to remove the broken
+ echo amavis user, or repair it so that it is sane
amavis user, or repair it so that it is sane
+ exit 1
dpkg: error processing package amavisd-new (--configure):
 subprocess installed post-installation script returned error exit status 1
Errors were encountered while processing:
 amavisd-new
E: Sub-process /usr/bin/dpkg returned an error code (1)
Error: /Stage[main]/Amavisd::Install/Package[amavisd-new]/ensure: change from absent to present failed: Execution of '/usr/bin/apt-get -q -y -o DPkg::Options::=--force-confold install amavisd-new' returned 100: Reading package lists...
```

**Comment by coreone** [comment]
@rgevaert Sorry...I wasn't watching this repo, so this issue got by me.  I think this issue should be resolved now with PR #2. Let me know if anything else needs to be done to fix this for Debian.

**Comment by rgevaert** [comment]
Thanks I'll check it!
